dbt/parsely_dpl/run_parsely_dpl.sh

```diff
@@ -0,0 +1 @@
+dbt run --models base.*+ --profiles-dir /home/annelise/.dbt/
```

Comment:
Maybe we could change this to $HOME, depending on how .dbt stuff installs (not sure about that part).